src/main/scala/in/rcard/http4s/tutorial/movie/MovieApp.scala

```diff
@@ -86,33 +88,33 @@ object MovieApp {
     }
   }
 
-  def directorRoutes[F[_] : Sync]: HttpRoutes[F] = {
+  def directorRoutes[F[_] : Async]: HttpRoutes[F] = {
     val dsl = new Http4sDsl[F] {}
     import dsl._
-    implicit val directorDecoder: EntityDecoder[F, Director] = jsonOf[F, Director]
+    implicit val directorDecoder: EntityDecoder[F, Director] = jsonOf[F, Director] // most calls require Sync, but this requires Concurrent => require Async
     import cats.syntax.flatMap._
     import cats.syntax.functor._
     HttpRoutes.of[F] {
       case GET -> Root / "directors" / DirectorVar(director) =>
         if (director == Director("Zack", "Snyder"))
-          Ok(Director("Zack", "Snyder").asJson, Header("My-Custom-Header", "value"))
+          Ok(Director("Zack", "Snyder").asJson, Header.Raw(CIString("My-Custom-Header"), "value"))
```

Comment:
Caused by API change

Reply:
Mmm...I don't think they really improve the syntax with this change. Thanks :)